src/lib.rs

```diff
@@ -42,7 +42,7 @@ const BACKEND_DIR: &Path = path!("se050-bak");
 pub const GLOBAL_ATTEST_ID: ObjectId = ObjectId(hex!("F0000012"));
 
 /// The version to know wether it should be re-configured
-pub const SE050_CONFIGURE_VERSION: u32 = 2;
+pub const SE050_CONFIGURE_VERSION: u32 = 3;
```

Comment:
Why do we need a new version?  AFAIS `REQUIRED_CURVES` did not change.

Reply:
This PR removes the `large-ecc` feature  so they're now always enabled. Worst case I don't think it's a problem to bump the version.

Reply:
But the feature did not affect curve initialization.  See also:  https://github.com/Nitrokey/trussed-se050-backend/pull/33#discussion_r1698125710

Reply:
Right, removed.